lib/node_modules/@stdlib/iter/cartesian-square/test/test.js

```diff
@@ -0,0 +1,91 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var iterCartesianSquare = require( './../lib/main.js' );
+
+
+// TESTS //
```

Comment:
This tests do not have full coverage support. You can run

```bash
make test-cov TESTS_FILTER=".*/iter/cartesian-square/.*"
make view-cov
```

to view a coverage report. See other iterator packages for how we ensure that tests can run in older environments lacking Symbols.